plugins/blame.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: blame
+spec:
+  version: v0.0.2
+  platforms:
+  - bin: blame
+    uri: https://github.com/knight42/kubectl-blame/releases/download/v0.0.3/kubectl-blame-v0.0.3-linux-amd64.tar.gz
+    sha256: e6fbad9f6dbd8b9847c57e203858d289be11e3f3eb51d735d84fba2ff2736efe
+    files:
+      - from: ./LICENSE
+        to: .
+      - from: ./kubectl-blame
+        to: blame
+    selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+  - bin: blame
+    uri: https://github.com/knight42/kubectl-blame/releases/download/v0.0.3/kubectl-blame-v0.0.3-darwin-amd64.tar.gz
+    sha256: 617c81dce51da5c05615054a759c6ca22aa63165fde6479146a7e5d785f9dad9
+    files:
+      - from: ./LICENSE
+        to: .
+      - from: ./kubectl-blame
+        to: blame
+    selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+  shortDescription: Annotate each line in the given resource's YAML with information from the managedFields to show who last modified the field.
```

Comment:
This is not short at all. It won't show most of it. Please look at other plugins.